### PR TITLE
Revert "makefile: add fusion bases required to generate fusion bundle"

### PR DESCRIPTION
**Pull request by iamniting**
This reverts commit c9232375d5f57f67b13d3bb7272c46b7ff3ed10c.

Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: config/manifests/kustomization.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
-
```
Comment (malayparida2000) [inline config/manifests/kustomization.yaml:3]: Why are we removing the apiVersion & Kind, Aren't they necessary?

Comment (iamniting) [reply config/manifests/kustomization.yaml:3]: no, they are not. This is how the scaffolding was created earlier. As it is revert git is reverting everything. They get populated if and only if we change the file. Which was the case when we added the fusion.

Comment (malayparida2000) [reply config/manifests/kustomization.yaml:3]: ack

**Review by malayparida2000 — APPROVED** [review]
/lgtm

**Pull request merged**